redbot/core/core_commands.py

```diff
@@ -443,6 +468,407 @@ async def uptime(self, ctx: commands.Context):
+            for ext in no_statements:
+                parts.append(f"\n - {entity_transformer(ext)}")
+
+        pretty_things = (  # This ends up being a small bit extra that really makes a difference.
+            """
+            <!DOCTYPE html>
+            <html>
+            <head><meta charset="utf-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+            <title>3rd Party Data Statements</title>
+            <style type="text/css">
+            body{margin:2em auto;max-width:800px;line-height:1.4;font-size:16px;
+            background-color=#EEEEEE;color:#454545;padding:1em;text-align:justify}
+            h1,h2,h3{line-height:1.2}
+            </style></head><body>
+            """
+        )
+
+        generated = markdown.markdown("\n".join(parts), output_format="html")
+
+        closing = "</body></html>"
+
+        html = "\n".join((pretty_things, generated, closing))
+
+        fp = io.BytesIO(html.encode())
+
+        await ctx.send(file=discord.File(fp, filename="3rd-party.html"))
+
+    async def get_serious_confirmation(self, ctx: commands.Context, prompt: str) -> bool:
+
+        confirm_token = "".join(random.choices((*ascii_letters, *digits), k=8,))
+
+        await ctx.send(f"{prompt}\n\n{confirm_token}")
+        try:
+            message = await ctx.bot.wait_for(
+                "message",
+                check=lambda m: m.channel.id == ctx.channel.id and m.author.id == ctx.author.id,
+                timeout=30,
+            )
+        except asyncio.TimeoutError:
+            await ctx.send(_("Did not get confirmation, cancelling."))
+        else:
+            if message.content.strip() == confirm_token:
+                return True
+            else:
+                await ctx.send(_("Did not get a matching confimration, cancelling."))
+
+    # 1 per day, not stored to config to avid this being more stored data.
+    # large bots shouldn't be restarting so often that this is an issue,
+    # and small bots that do restart often don't have enough
+    # users for this to be an issue.
+    @commands.cooldown(1, 86400, commands.BucketType.user)
+    @mydata.command(cls=commands.commands._AlwaysAvailableCommand, name="forgetme")
+    async def mydata_forgetme(self, ctx: commands.Context):
+        """
+        Have [botname] forget what it knows about you.
+
+        This may not remove all data about you, data needed for operation,
+        such as command cooldowns will be kept until no longer neccessary.
+
+        Further interactions with [botname] may cause it to learn about you again.
+        """
+        if ctx.assume_yes:
+            # lol, no, we're not letting users schedule deletions every day to thrash the bot.
+            ctx.command.reset_cooldown(ctx)  # We will however not let that lock them out either.
+            return await ctx.send(_("This command does not support non-interactive usage"))
```

Comment:
You *might* want to include the command name in this message.

Reply:
I was being lazy with the translation handling here, but I've gone and added that for clarity in case other coammands decide to vocally complain about being scheduled.